lips/lip-8.md

```diff
@@ -17,14 +17,25 @@ An extension of the Off-Chain protocol to provide support for more advanced merc
 
 Version 0 of the Off-Chain Protocol is described in [LIP 1](https://lip.libra.org/lip-1/).  Version 1 as described here is an extension of the Off-Chain Protocol and adds features to support more advanced functionality - particularly targeted to support merchant use-cases.  This is inclusive of pull payments, recurring payments, and auth/capture flows.
 
+---
+# Disclaimer
+---
+
+This LIP does not contain the initial phase of a sub-account discovery that is required to start negotiating merchant scenarios.
+The process describes below starts at the phase that both sides (particularly the biller side) have the relevant subaddresses. 
+For the discovery phase, there are currently two leading methods:
+  1. The buyer VASP triggers a process with pre-knowledge of the merchant subaddress. e.g., QR/deep links - where the consumers while acting in their wallet provides their VASP the merchant's relevant details (embedded in the QR/link in the checkout page)
+  2. The merchant gets the user identifier (like Pay ID) and query the buyer side for a new subaddress for this process
+For now, we will assume that one of these methods took place and the merchant have the buyer subaddress at hand.
+  
```

Comment:
I think we should leave this portion out.  It's outside the scope of this LIP and I think it's enough to mention that it isn't in the scope of this LIP without speculating how it may be done

Reply:
Agreed. I will remove it.